xbmc/windowing/WinSystem.h

```diff
@@ -85,6 +85,21 @@ class CWinSystemBase
   virtual bool UseLimitedColor();
   //the number of presentation buffers
   virtual int NoOfBuffers();
+  /**
+   * Get average display latency
+   *
+   * The latency should be measured as the time between finishing the rendering
+   * of a frame, i.e. calling PresentRender, and the rendered content becoming
+   * visible on the screen.
+   *
+   * \return average display latency in seconds, or negative value if unknown
+   */
+  virtual float GetDisplayLatency() { return -1.0f; }
+  /**
+   * Get time that has elapsed since the start of the rendering the current frame
+   * in seconds
```

Comment:
AdvancedSettings::GetDisplayLatency is also in seconds - shouldn't it be consistent?